crates/dbsp/src/dynamic/lean_vec.rs

```diff
@@ -1455,8 +1455,8 @@ impl<T: SizeOf> SizeOf for LeanVec<T> {
     fn size_of_children(&self, context: &mut Context) {
         self.vec.size_of_children(context, &mut |x, ctx| {
             let x = unsafe { &*(x as *const T) };
-            x.size_of_children(ctx);
-        })
+            T::size_of_children(x, ctx);
```

Comment:
I am not sure we have properly implemented this trait everywhere. We should make an audit